saleor/graphql/schema.graphql

```diff
@@ -380,6 +380,41 @@ type AppUpdate {
   app: App
 }
 
+enum AreaUnitsEnum {
+  SQ_CHAIN
+  SQ_CHAIN_BENOIT
+  SQ_CHAIN_SEARS
+  SQ_BRITISH_CHAIN_BENOIT
+  SQ_BRITISH_CHAIN_SEARS
+  SQ_BRITISH_CHAIN_SEARS_TRUNCATED
+  SQ_CM
+  SQ_BRITISH_FT
+  SQ_BRITISH_YD
```

Comment:
I think we can keep only generic "feet" and "yards" without specifying special variants. The goal here is to provide the necessary units, not to convert between them accurately.